src/main/java/duke/DukeHistory.java

```diff
@@ -0,0 +1,266 @@
+package duke;
+
+import java.lang.StringBuilder;
+
+import java.util.ArrayList;
+
+/**
+ * Represents a class that maintains an ArrayList<Task> records using a variety of methods, that is created
+ * at the beginning of each session of Duke.
+ */
+public class DukeHistory {
+    private final ArrayList<Task> records = new ArrayList<>(100); // ArrayList of size 100 by default
+
+    public DukeHistory() { //Empty Constructor
+    }
+
+    /**
+     * A method that, when called, attempts to initialize a ToDos task using the inputted description and adds it to the
+     * ArrayList<Task> records.
+     *
+     * @param description A String description interpreted by Commands.todo().
+     */
+    public void addToDo(String description) {
+        ToDos tempToDo = new ToDos(description);
+        records.add(tempToDo);
+        String msg = "_______________________________________________________\n"
+                + "Understood, adding this task now:\n"
+                + "    " + tempToDo.getToDo()
+                + "Currently you have " + records.size() + " tasks in our records.\n"
+                + "_______________________________________________________\n";
+        System.out.println(msg);
```

Comment:
Should this be abstracted into `DukeUi.java`? This applies to all `String msg` in this file. 😄